src/server/graphql/models/CachedUser/cachedUserSchema.js

```diff
@@ -122,7 +122,7 @@ export const CachedUser = new GraphQLObjectType({
       type: new GraphQLList(TeamMember),
       description: 'The memberships to different teams that the user has',
       async resolve({id}) {
-        return await r.table('TeamMember').getAll(id, {index: 'cachedUserId'});
+        return await r.table('TeamMember').getAll(id, {index: 'userId'});
```

Comment:
refactored `cachedUserId` to just `userId` throughout, since `userId` is the PK for both `CachedUser` and `UserProfile`.